src/test/java/chess/repository/dao/PieceDaoTest.java

```diff
@@ -43,51 +42,35 @@ void setUp() {
     @DisplayName("진행된 게임에서 체스 보드를 읽어온다.")
     void findChessBoardByGameIdTest() {
         // given
-        int gameId = gameRepository.save(chessGameManager);
+        long gameId = gameRepository.save(chessGameManager, "test title");
 
         // when
         pieceDao.savePieces(chessGameManager, gameId);
         ChessBoard chessBoardFound = this.pieceDao.findChessBoardByGameId(gameId);
 
         // then
-        assertThat(chessGameManager.getBoard()).isEqualTo(chessBoardFound);
+        assertThat(chessBoardFound).isEqualTo(chessGameManager.getBoard());
     }
 
     @Test
     @DisplayName("진행되고 있는 게임의 모든 기물을 저장한다.")
     void savePiecesTest() {
         // given
-        int gameId = gameRepository.save(chessGameManager);
+        long gameId = gameRepository.save(chessGameManager, "test title");
 
         // when
         pieceDao.savePieces(chessGameManager, gameId);
 
         //then
-        Integer rowFound = this.jdbcTemplate.queryForObject("SELECT count(*) FROM piece WHERE game_id = " + Integer.toString(gameId), Integer.class);
+        Integer rowFound = this.jdbcTemplate.queryForObject("SELECT count(*) FROM piece WHERE game_id = " + Long.toString(gameId), Integer.class);
```

Comment:
```suggestion
        Integer rowFound = this.jdbcTemplate.queryForObject("SELECT count(*) FROM piece WHERE game_id = ?", Integer.class, gameId);
```

비록 테스트 코드라 하더라도 SQL 문 생성 시 문자열을 더하는 건 지양해 주세요!

Reply:
앗😨!! 네!! [수정](https://github.com/tributetothemoon/jwp-chess/commit/31a19ca7fb46e0fe6a10932c1c56f25179539961)하였습니다!!